docs/api/blocks.md

```diff
@@ -285,8 +287,10 @@ GET /api/v2/blocks/{id|height}
 curl https://api.nos.dev/api/v2/blocks/15447090855222023348
```

Comment:
I don't know where this example comes from but what I get as `id` is a significantly longer base 16 string.